src/components/BrandConfigProvider/styles/createStyles.ts

```diff
@@ -0,0 +1,54 @@
+import { ImageStyle, TextStyle, ViewStyle } from 'react-native';
+import { Theme } from '../theme/ThemeProvider';
+
+/**
+ *
+ * @param namespace A string literal name that becomes the first level property names of
+ * the style object returned from `useStyles` - usually the exact name of the component being styled.
+ * @param styleBuilder This is a function that takes a theme and returns an object with named styles.
+ * Each named styles will be typed according to the case-insensitive suffix of their name:
+ *   - Names ending in `Image` will become an `ImageStyle` type
+ *   - Names ending in `Text` or `Label` will become a `TextStyle` type
+ *   - Names ending in anything else (often `View` or `Container`) will become a `ViewStyle` type
```

Comment:
Not sure if it would be worth stating that we can expand this if we find there are other suffixes that might make sense.

Reply:
Updated.